contents/blog/how-come-we-ship-so-much.md

```diff
@@ -0,0 +1,134 @@
+
+One of the most common questions I get asked is how do we manage to build so much, so quickly.
+
+## We designed the company around speed
+
+We provide an all-in-one platform to help engineers build better products. We have a list of [25+ products](../handbook/which-products) we want to build.
+
+That means we had to design our company primarily for speed.
+
+## These ideas may _suck_ for you
+
+Before you start making changes, think hard about what your product should be optimized for.
+
+For example, if you are shipping the latest iPhone, complete design control is _more_ important to your users than having all the features. Likewise, if you are building software and are building a tiny product that has better UX than an incumbent, design and control matters.
+
+I _suspect_ that you could still get higher quality with more autonomy – I'd probably have lots of designers and engineers collaborating with relatively few product managers, but I can't speak to the above as confidently since my experience is building a multi-product platform.
+
+## No product management by default
+
+The responsibilities of a product manager still exist, but our engineers:
+
+* Talk to users.
+* Decide what to build within their product (and sometimes pitch whole new products).
+* Have complete access to metrics like revenue, hosting costs, and (soon) staff costs, for their product.
+
+The concept internally is a "[product engineer](/blog/what-is-a-product-engineer)" – i.e. an engineer who is product-minded and autonomous, rather than filing tickets mindlessly.
+
+We have one product manager who steps in reactively if a small team needs more support.
+
+The above _requires_ a lot of context setting from the company. We are _unusually transparent_ – we share board slides, financial information, fundraising progress before, during and after it happens, and much more. 
+
+
+## No design by default
+
+When we started, we felt that if product designers dictate to engineers, the best engineers would eventually quit as they'd want more freedom.
+
+As we grew, we realized that engineers _can_ design the UX of a product. Especially if we hire people that have this skillset.
+
+We invested quite a lot in getting our design _system_ up and running, and worked with a designer to get it done. Then we moved onto this "no design by default" phase. Had we been in this phase from the beginning, the app would have been very inconsistent... at best.
+
+Today, we have people that can help engineers with design, but they are very happy to work reactively and fast – and they don't block merging work during QA. This means we _do_ ship work where the design isn't perfect, but we won't drop things that are important and will keep improving them.
+
+For example, we've redesigned this entire website ~5 times.
+
+## Hiring is key
```

Comment:
I feel like there might be a better title here? Something like "Hiring experience and/or ambition". Something that describes our approach, rather than just saying "it's important".